src/ShapeContourDrawer.js

```diff
@@ -69,15 +69,20 @@
 import { ContourBuilder } from './ContourBuilder';
 
 export class ShapeContourDrawer {
-  constructor(levels) {
+  constructor(levels, swapAxes) {
     this.contours = new Array(levels.length);
     for (var i = 0; i < levels.length; i++) {
       this.contours[i] = new ContourBuilder(levels[i]);
     }
+    this.swapAxes = swapAxes;
   }
 
   drawContour(x1, y1, x2, y2, z, k) {
-    this.contours[k].addSegment({ x: x1, y: y1 }, { x: x2, y: y2 });
+    if (!this.swapAxes) {
+      this.contours[k].addSegment({ x: y1, y: x1 }, { x: y2, y: x2 });
```

Comment:
The 2 lines of the conditions are equal. Not sure which one should be changed.

Reply:
I already fixed it in https://github.com/mljs/conrec/pull/6/commits/0e319b258033c0f95538c749906e290d56aee0b4